packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-dart-visitor.test.ts

```diff
@@ -15,12 +15,13 @@ const getVisitor = (
   enableDartNullSafety: boolean = false,
   enableDartNonModelGeneration: boolean = true,
   isTimestampFieldsAdded: boolean = false,
+  emitAuthProvider: boolean = false
```

Comment:
What would be the case we don’t want to emit them?

Reply:
For all existing projects the feature flag will be set as false, for all new projects this feature flag value is set as true per amplify-ios library maintainers' input.

Reply:
Hmm I think iOS's situation was a bit different. I would prefer to have it always on, but I will default to your judgment since you had the conversations with them.